Linear_cell_complex/include/CGAL/Linear_cell_complex_constructors.h

```diff
@@ -339,9 +339,9 @@ namespace CGAL {
           else if ( it->vertex(1) == atr.infinite_vertex() )
             dart = alcc.beta(res, 1);
           else if ( it->vertex(2) == atr.infinite_vertex() )
-            dart = alcc.beta(res, 2);
+            dart = alcc.beta(res, 1, 1);
```

Comment:
But you can see in [this code](https://gist.github.com/pranavkantgaur/0cf18af7f21c76f82643#file-testbeta-cpp) they return coordinates of different vertices.